microsoft/testsuites/vm_extensions/azure_keyvault_vm_extension.py

```diff
@@ -63,7 +63,7 @@ def _check_system_status(node: Node, log: Logger) -> None:
         expected_exit_code_failure_message=message,
     )
 
-    certb = "/var/lib/waagent/a/symbolinkb"
+    certb = "/var/lib/waagent/b/symbolinkb"
```

Comment:
Why is this char changed? Is it possible to be changed in future? Should we use a wildchar to match more paths?

Reply:
it is a test bug, it was written wrong in the beginning.
![image](https://github.com/user-attachments/assets/558d2c17-6b2e-46b7-b80b-cea1d4f1ab87)

Reply:
Should it support both a and b?

Reply:
updated